src/components/stacks/stacks-directive.tpl.html

```diff
@@ -7,11 +7,11 @@
           <input type="checkbox" ng-click="vm.updateSelection()" ng-checked="vm.selectedIds.length > 0" ng-disabled="vm.stacks.length === 0"><i></i>
         </label>
       </th>
-      <th>Summary</th>
-      <th class="percentage">Users</th>
-      <th class="number">Count</th>
-      <th class="date hidden-xs">First</th>
-      <th class="date hidden-xs">Last</th>
+      <th>{{'Summary' | translate}}</th>
+      <th class="percentage">{{'UserRatio' | translate}}</th>
```

Comment:
Should be Users

Reply:
I set this `Users` to `UserRatio`.Because Users have two meaning `count` and `ratio`. In `Sessions Dashboard` is count.In `Other Dashboard` is ratio.

Reply:
@Varorbc looking at this again, We'd feel more comfortable if this and a few other changes were reverted (we can talk about them more in a new issue).